### PR TITLE
Add TravisCI support.

### PR DESCRIPTION
**Pull request by miguelandres**
-Add readme codecov & status badges.
-De-linting so it doesn't fail on its first run.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - ANDROID_TARGET=android-24
+
+android:
+  components:
+    - platform-tools
+    - tools
+    - android-24
+    - build-tools-24.0.1
+    - extra-android-support
+    - extra-android-m2repository
```
Comment (pingpongboss) [inline .travis.yml:18]: can you verify that you need these 2 `extra-android-` components? See if travsici builds without them.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+licenses:
+  - 'android-sdk-license-.+'
+
+before_install:
+  - echo yes | android update sdk --filter extra-android-m2repository --no-ui --force > /dev/null
+
+script:
+  - ./gradlew build test
```
Comment (pingpongboss) [inline .travis.yml]: I suggest using `check` which includes `build` and `test` but also includes plugin outputs like jacoco (for code coverage).

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/material-foundation/material-remixer-android.svg?branch=develop)](https://travis-ci.org/material-foundation/material-remixer-android) [![codecov](https://codecov.io/gh/material-foundation/material-remixer-android/branch/develop/graph/badge.svg)](https://codecov.io/gh/material-foundation/material-remixer-android)
```
Comment (pingpongboss) [inline README.md]: we have codecov on a separate line (in source) which helps readability

**Review by pingpongboss — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (miguelandres) [reply .travis.yml:18]: Android support is definitely necessary.

File: README.md (hunk 3 above)
Comment (miguelandres) [reply README.md]: done

File: .travis.yml (hunk 2 above)
Comment (miguelandres) [reply .travis.yml]: done

**Review by pingpongboss — APPROVED** [review]

**Comment by miguelandres** [comment]
Finally figured it out! Travis installs old tools by default, and you have to work around it by installing them twice.

https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943

I'll be squashing this PR, since the history is basically useless, a bunch of attempts to diagnose the issue.

**Review by pingpongboss — COMMENTED** [review]
Hmm think I should do this for MM?

**Comment by miguelandres** [comment]
Probably...it may be that your target api is 23 and it hasn't affected you
but it would if you move to 24/25

Den tors 10 nov. 2016 12:56Mark Wei notifications@github.com skrev:

**Pull request merged**